modules/ROOT/pages/keycloak/configuring-dev-env-keycloak.adoc

```diff
@@ -0,0 +1,18 @@
+include::{partialsdir}/attributes.adoc[]
+
+:service-name: Identity Management
+
+= Configuring your Development Environment for the {service-name} Service
+
+== Prerequisites
+
+* [x] You have provisioned one of the {product-name} as described in xref:keycloak/provisioning.adoc[Provisioning {service-name} Service].
+* [x] (Optional) You have associated the service with a {mobile-client} as described in xref:keycloak/binding.adoc[Binding a {mobile-client} with the {service-name} Service].
+
+NOTE: You may skip creating a binding if only interested in metrics from other services but not from a {mobile-client}.
```

Comment:
What this means? I do not understand that

Reply:
So, this PR is all about a pattern being applied to the docs. A part of this work was to apply generic files across the different services - ie: get the structure in place. As evident by this question, some non generic info was brought with the generic files.

In summary, this looks like specific 'metrics' content - I'll pull it out of here unless you think the following would make sense:

"+NOTE: You may skip creating a binding if only interested in Identity Management from other services but not from a {mobile-client}."

Reply:
@jstaffor I do not understand this sentence, but I'm fine anyway.